app/csv/Charge.scala

```diff
@@ -68,6 +68,25 @@ Charge_day // day we're running for
      and RatePlanCharge.Name = 'Holiday Credit')
     */
 
+  val Rate_Plan_Charge__Charged_Through_Date = 0
```

Comment:
i had a merge conflict and assumed that they needed to be there. if you don't need them please delete them again. sorry. (this is exactly what you were talking about before :)